nixos/modules/services/misc/linux-enable-ir-emitter.nix

```
@@ -0,0 +1,62 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+with lib; let
```

```suggestion
let
```
to address #208242 for this module, uses of `with` such as this should be avoided.
consider `inherits` where you find yourself using `lib.foo` excessively.